docs/blog/20230701-Daeraxa-JulyUpdate.md

```diff
@@ -0,0 +1,127 @@
+
+However this does mean you can now easily manage it. For example, once you have installed `deb-get` all you have to do to install Pulsar is:
+
+- `deb-get install pulsar`
+
+To update Pulsar (and other `deb-get` packages, all you need to do is:
+
+- `deb-get update`
+- `deb-get upgrade`
+
+So if you are on a Debian based distro and want a quick and easy way to install and update Pulsar, give it a try!
+
+## Tree-sitter migration of TOML grammar
+
+<img src="./assets/toml.png" height=200>[^3]
+
+In our [last regular release](https://pulsar-edit.dev/blog/20230616-Daeraxa-v1.106.0.html) we made a big deal about our modern Tree-sitter migration and new grammars that come with it.
+
+This work is still very much ongoing and we have new grammars being migrated all the time - our latest is the [TOML Tree-sitter grammar](https://github.com/pulsar-edit/pulsar/pull/617) that [@savetheclocktower] has added which come with all the benefits of a Tree-sitter grammar such as more consistent highlighting and scope for new features in the future.
+
+You can try this out in our [rolling release](https://pulsar-edit.dev/download.html#rolling-release) or wait until the next regular one. Either way you will need to tick the `Use Modern Tree-Sitter Implementation` option in your `Core` settings in order to enable it.
+
+As ever we are after feedback, so if you use this new TOML Tree-sitter grammar (or any other for that matter) and have any issues or see any inconsistencies compared to the existing grammar then please let us know on [Discord](https://discord.gg/7aEbB9dGRT) or [file an issue](https://github.com/pulsar-edit/pulsar/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc).
+
+## `less-cache` package update
+
+<img src="./assets/less.png" height=200>[^4]
+
+This is still in the works but [@confused-techie] has been making some updates to be added in the near future that bumps the version of `less-cache` that we are using.
+
+`less-cache` is a module that handles turning all Pulsar's (and our community's) packages written in [less](https://lesscss.org/) into valid CSS, as well as ensuring each one is able to import the values from Pulsar that it cares about. For the past three years `less-cache` has been using `less@3.12.2` which, while fine, has been missing out on some of the big new changes in less after version 4 was released.
+So [this PR](https://github.com/pulsar-edit/pulsar/pull/611) bumps it to `less-cache@2.0.0` in order to bump the version of `less` to 4.1.3 which brings along these exciting new features but also introduces breaking changes to existing `less` stylesheets. The biggest of which being:
+
+- Parantheses required for mixin calls
+- Parens-division now the default math setting
```

Comment:
```suggestion
- Parentheses required for mixin calls (`.some-mixin()` instead of `.some-mixin`)
- Parentheses now required around math expressions that use division
```

Reply:
Yeah the typo here is my bad, so whoops!

But I do want to add the other than the typo, this message is identical to what was included on the [`v4.0.0` release notes](https://github.com/less/less.js/releases/tag/v4.0.0), so things like `Parens-division` don't have to be changed, but if we feel they are more understandable changed, then that's totally fair

Reply:
Yeah if it is on the release notes it should be retained here (also my british brain just wants to get rid of "parentheses" entirely but I know this should be left :P )